packs/astral/actions/get_sunrise.yaml

```diff
@@ -0,0 +1,6 @@
+---
+name: "get_sunrise"
+runner_type: "run-python"
+description: "Returns the time of the sunrise in UTC"
```

Comment:
For convenience it would also be nice if user could supply a timezone / location (as a config option or action parameter or both).